packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -7,6 +7,7 @@ import {
   getDirectoryPath,
   joinPaths,
   logDiagnostics,
+  NoTarget,
```

Comment:
Where is this used?

Reply:
When we create diagnostic and there is no target node, we will specify `NoTarget`. e.g.